### DISCUSSION TITLE
Suggestions for a Cleaner Layout and Improved navigation? Experience

### DISCUSSION THREAD

**Discussion by Nicknamely** [zen-browser/desktop#3730] [Ideas]
![image](https://github.com/user-attachments/assets/d098f474-08b3-414e-ad04-bb77ce87fa16)

### Search Bar and URL Bar Overlap:
When I enable the search bar on the homepage, both the URL bar and the sidebar show up at the same time. This makes the layout feel cluttered and a bit confusing to use.

### Floating URL Bar Customization:
The floating URL bar is useful sometimes, but other times, the sidebar alone does the job perfectly. Right now, there’s no way to turn off the floating URL bar or hide it when it’s not needed.

### Mobile-like Scrolling Behavior:
It would be awesome if the page reloaded when I scroll up while already at the top—just like other browsers. It’s a small thing, but it makes navigation feel much smoother.

### Simplify the Search and URL Bar Layout:
When enabling the search bar, maybe only show one interface at a time (either the URL bar or the sidebar, not both). It’ll make things look cleaner and easier to use.

### Add Options for the Floating URL Bar:
Could you add an option in settings to disable the floating URL bar for people who prefer the sidebar?
It’d also be great to have Ctrl+L to quickly toggle the URL bar on and **off** when needed.

### Make Scrolling Up Reload the Page like on mobile (for laptops only ?)
Adding a "pull-to-refresh" style reload when scrolling up at the top of a page.s

**Comment by Seele0oO** [comment]
I think that adding a close button to the ‘Floating URL Bar’ would be very effective, because right now clicking on a link doesn't edit where you clicked on it, which is a bit of a cut-throat experience.

As for pull-to-refresh, I think it's still too aggressive, if you mean scroll to the top and keep scrolling, then it's probably more of a hassle than clicking Refresh or F5, and if you mean dragging the mouse down, it conflicts with most web page operations.
